[2024-08-16T10:00:00.554+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-16T10:00:00.719+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Weather_Data_Extraction.no_new_data backfill__2024-08-11T00:00:00+00:00 [queued]>
[2024-08-16T10:00:00.729+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Weather_Data_Extraction.no_new_data backfill__2024-08-11T00:00:00+00:00 [queued]>
[2024-08-16T10:00:00.730+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-16T10:00:00.745+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): no_new_data> on 2024-08-11 00:00:00+00:00
[2024-08-16T10:00:00.752+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1985) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-16T10:00:00.754+0000] {standard_task_runner.py:63} INFO - Started process 1987 to run task
[2024-08-16T10:00:00.754+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Weather_Data_Extraction', 'no_new_data', 'backfill__2024-08-11T00:00:00+00:00', '--job-id', '1225', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/Data_Weather_Extraction.py', '--cfg-path', '/tmp/tmp1j09564k']
[2024-08-16T10:00:00.757+0000] {standard_task_runner.py:91} INFO - Job 1225: Subtask no_new_data
[2024-08-16T10:00:00.814+0000] {task_command.py:426} INFO - Running <TaskInstance: Weather_Data_Extraction.no_new_data backfill__2024-08-11T00:00:00+00:00 [running]> on host ubuntu
[2024-08-16T10:00:00.910+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ducky' AIRFLOW_CTX_DAG_ID='Weather_Data_Extraction' AIRFLOW_CTX_TASK_ID='no_new_data' AIRFLOW_CTX_EXECUTION_DATE='2024-08-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2024-08-11T00:00:00+00:00'
[2024-08-16T10:00:00.912+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-16T10:00:00.924+0000] {logging_mixin.py:188} INFO - Data are inserted, skipping this run
[2024-08-16T10:00:00.924+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-16T10:00:00.925+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-16T10:00:00.934+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Weather_Data_Extraction, task_id=no_new_data, run_id=backfill__2024-08-11T00:00:00+00:00, execution_date=20240811T000000, start_date=20240816T100000, end_date=20240816T100000
[2024-08-16T10:00:00.969+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-16T10:00:00.998+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-16T10:00:01.003+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
